sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClientIntegrationTest.java

```diff
@@ -96,6 +100,31 @@ static void cleanup() {
             .forEach(property -> client.deleteRule(topicName, subscriptionName, property.getName()));
     }
 
+    @Test
+    /**
+     * Test to connect to the service bus with an azure sas credential.
+     * ServiceBusSharedKeyCredential doesn't need a specific test method because other tests below
+     * use connection string, which is converted to a ServiceBusSharedKeyCredential internally.
+     */
+    void azureSasCredentialsTest() {
```

Comment:
nit: move the @Test annotation to the middle of comments and method for consistency.